spec/abide_dev_utils/xccdf_spec.rb

```diff
@@ -38,6 +38,10 @@
     expect { AbideDevUtils::XCCDF::Benchmark.new(windows_xccdf) }.not_to raise_error
   end
 
+  it 'genmap' do
```

Comment:
RSpec test names are supposed to read like english and they should start with a verb. Something like `generates a map data structure successfully` would work here.

Reply:
Yeah, this doesn't even work yet.